### DISCUSSION TITLE
Is it possible to create a wildcard regex to redirect certain terms?

### DISCUSSION THREAD

**Discussion by 88d-963** [mshibanami/redirect-web#14] [Questions] [answered]
I was wondering if it is possible to create a rule whereby if a certain word appears in any part of the url the redirect occurs.

e.g. the url contains the word "youtube"

**Comment by mshibanami** [comment, marked as answer]
@88d-963 Yes, you can by setting `*youtube*` as a Wildcard pattern to the Redirect From section, like this:
  
![Screenshot 2023-11-01 at 12 56 40 pm](https://github.com/mshibanami/redirect-web/assets/1333214/ed8bd8cf-1e6d-44a0-97b9-5baaf67a96b5)

You can find the details here:
https://mshibanami.github.io/redirect-web/#/redirect-rule?id=url-pattern
